test/requiregSpec.js

```diff
@@ -52,15 +54,15 @@ describe('requireg', function () {
 
       it('should have the expected module path', function () {
         expect(requiregModule.resolve('beaker'))
-          .to.be.equal(__dirname + '/fixtures/lib/node_modules/beaker/index.js')
-      }) 
+          .to.be.equal(path.join(__dirname, 'fixtures', 'lib', 'node_modules', 'beaker', 'index.js'))
```

Comment:
Hi @mhofman,
Just FYI, as far as I remember you can put `fixtures/lib/node_modules/beaker/index.js` here and `path.join` will take care to normalize it (passing multiple params is not a crime though, matter of personal taste)

https://nodejs.org/api/path.html#path_path_join_path

> The path.join() method join all given path segments together using the platform specific separator as a delimiter, then normalizes the resulting path.

Other than that LGTM :+1: